src/pages/Settings.vue

```diff
@@ -121,6 +147,43 @@ export default {
                 },
             };
         },
+
+        breadcrumbs: ({ $route, subMenus }) => {
+            // List of setting routes matching the current path
+            // for ex, with path "/settings/users/edit/1", we got:
+            // [
+            //    { path: "/settings/users", ...otherRoutesProps },
+            //    { path: "/settings/users/edit/:id", ...otherRoutesProps }
+            // ]
+            const settingRoutes = uniqBy(
+                $route.matched.filter(({ path }) => /^\/settings\//.test(path)),
+                "path"
+            );
+
+            /** Get leaf submenu for a given setting path
+            * @param {string} path setting path, ex: "/settings/users/edit/1"
+            * @returns {Object} the leaf subMenu corresponding to the given path
+            */
+            const getLeafSubMenu = path => {
+                const pathParts = path.split("/").slice(2);
+
+                // walk through submenus and there children until reaching the corresponding leaf subMenu
+                // ex with "/settings/users/edit/1" we got { title: this.$t("Edit") }
+                // because this path match "users" > "children" > "edit" in subMenus
+                return pathParts.reduce(
+                    (acc, pathPart) =>
+                        acc[pathPart] || acc.children?.[pathPart] || acc,
+                    subMenus
+                );
+            };
+
+            // construct an array of setting routes path joined with submenus
+            return settingRoutes.map(({ path }, idx) => ({
+                path, // the setting route path
+                title: getLeafSubMenu(path).title, // the corresponding submenu title
+                active: !settingRoutes[idx + 1] // is this breadcrumb the active route?
```

Comment:
I think these are self-explanatory => can be removed, otherwise much clearer 👍🏻 
```suggestion
                path,
                title: getLeafSubMenu(path).title,
                active: !settingRoutes[idx + 1],
```

Reply:
>  otherwise much clearer 👍🏻

Glad to read that! 👯‍♀️ 

> can be removed

Done 👍🏼